packages/nextjs/components/CheckinForm.tsx

```diff
@@ -19,7 +20,7 @@ const CheckinForm = ({ latLng = [0, 0] }) => {
     coordinateInputX: latLng[0], // to be picked up by prop
     coordinateInputY: latLng[1], // to be picked up by prop
     timestamp: nowInSeconds,
-    data: "Something you wanna add.",
+    data: "Memo.",
```

Comment:
This should be blank, with a placeholder attribute that says `Memo` with no period. But it should not be the default field — if people don't say anything, we don't want every log entry to say "Memo."